sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/TypesSamples.cs

```diff
@@ -0,0 +1,140 @@
+                .Types
+                .CreateOrReplaceAsync(timeSeriesTypes)
+                .ConfigureAwait(false);
+
+            // Check if the result error array does contain any object that is set
+            foreach (var result in createTypesResult.Value)
+            {
+                if (result.Error != null)
+                {
+                    Console.WriteLine($"Failed to create a Time Series Insights type: {result.Error.Message}.");
+                }
+            }
+            #endregion Snippet:TimeSeriesInsightsSampleCreateType
+
+            #region Snippet:TimeSeriesInsightsSampleGetTypeById
+            // Code snippet below shows getting a default Type using Id
+            Response<TimeSeriesTypeOperationResult[]> getTypeByIdResults = await client
+                .Types
+                .GetByIdAsync(new string[] { modelSettings.DefaultTypeId })
+                .ConfigureAwait(false);
+
+            TimeSeriesTypeOperationResult getTypeByIdResult = getTypeByIdResults.Value[0];
+
+            if (getTypeByIdResult.TimeSeriesType != null)
+            {
+                Console.WriteLine($"Retrieved Time Series Insights type with Id '{getTypeByIdResult.TimeSeriesType.Id}' and name '{getTypeByIdResult.TimeSeriesType.Name}'.");
+            }
+            else if (getTypeByIdResult.Error != null)
+            {
+                Console.WriteLine($"Failed to retrieve a Time Series Insights type with Id '{getTypeByIdResult.Error.Message}'.");
+            }
+            #endregion Snippet:TimeSeriesInsightsSampleGetTypeById
+
+            #region Snippet:TimeSeriesInsightsSampleReplaceType
+            // Update variables with adding a new variable
+            foreach (TimeSeriesType type in timeSeriesTypes)
+            {
+                type.Description = "Description";
+            }
+
+            Response<TimeSeriesTypeOperationResult[]> updateTypesResult = await client
+                .Types
+                .CreateOrReplaceAsync(timeSeriesTypes)
+                .ConfigureAwait(false);
+
+            // Check if the result error array does contain any object that is set
+            foreach (var result in createTypesResult.Value)
+            {
+                if (result.Error != null)
+                {
+                    Console.WriteLine($"Failed to replace a Time Series Insights type: {result.Error.Message}.");
+                }
+            }
+            #endregion Snippet:TimeSeriesInsightsSampleReplaceType
+
+            #region Snippet:TimeSeriesInsightsSampleGetAllTypes
+            // Get all Time Series types in the environment
+            AsyncPageable<TimeSeriesType> getAllTypesResponse = client.Types.GetTypesAsync();
+
+            await foreach (TimeSeriesType tsiType in getAllTypesResponse)
+            {
+                Console.WriteLine($"Retrieved Time Series Insights type with Id: {tsiType?.Id} and Name: {tsiType?.Name}");
```

Comment:
single quote around the id and name, similar to the other console print messages